app/src/main/java/com/example/marvelapp/api/mapper/CharacterMapper.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.marvelapp.api.mapper
+
+import com.example.marvelapp.api.response.DataResponse
+import com.example.marvelapp.entity.Character
+
+class CharacterMapper : BaseMapper<DataResponse, MutableList<Character>> {
```

Comment:
instead of using a class, this can be an object.. or better, an extension function

Reply:
done